site/en/install/source_windows.md

```diff
@@ -223,13 +248,14 @@ Although it is possible to build both CUDA and non-CUDA configs under the
 same source tree, we recommend running `bazel clean` when switching between
 these two configurations in the same source tree.
 
+
 ### Install the package
 
 The filename of the generated `.whl` file depends on the TensorFlow version and
 your platform. Use `pip3 install` to install the package, for example:
 
 <pre class="devsite-terminal tfo-terminal-windows prettyprint lang-bsh">
-pip3 install C:/tmp/tensorflow_pkg/tensorflow-<var>version</var>-cp36-cp36m-win_amd64.whl
+pip3 install C:/tmp/tensorflow_pkg/tensorflow-<var>version</var>-cp310-cp310-win_amd64.whl
```

Comment:
Is it possible to use a wild-card here so that the command to copy-paste doesn't depend on the python version.